influxdb3_server/src/http.rs

```diff
@@ -1670,6 +1671,26 @@ pub(crate) async fn route_request(
 ) -> Result<Response<Body>, Infallible> {
     let method = req.method().clone();
     let uri = req.uri().clone();
+
+    // Handle CORS Preflight Checks by allowing everything by default
+    // and allowing the check to be cached by the browser. This is useful
+    // for people wanting to query the DB directly from a browser rather
+    // than from a server. We're permissive about what works with CORS
+    // so we don't need to check the incoming request, just respond with
+    // the following headers. We do this before the API token checks as
+    // the browser will not send a request with an auth header for CORS.
+    if let Method::OPTIONS = method {
+        info!(?uri, "preflight request");
```

Comment:
Should this be `info`, might be noisy

Reply:
Ahh when I went to merge this comment showed up in my browser tab which I just had open for a few days. We could change it to debug. Wouldn't be a bad thing